death_cases_hashes.d.ts

```diff
@@ -0,0 +1,139 @@
+declare namespace RageEnums.Hashes {
+    const enum DeathCases {
+        weapon_electric_fence = 2461879995,
```

Comment:
We should have an unified constant naming. We should stick to uppercase constant names.